package.json

```
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "test": "eslint --ignore-path .gitignore --ignore-pattern _includes/readmes/ \"**/*.md\""
+    "test": "eslint . --ignore-path .gitignore --ignore-pattern _includes/readmes/ \"**/*.md\""
   },
   "devDependencies": {
     "eslint": "^8.54.0",
```

```suggestion
    "eslint": "^9.26.0",
```
[8.x EOL](https://eslint.org/version-support/)

---

Also update other linters to the newest version

---

@ShubhamOulkar would it make more sense to update the ESLint version and other plugins in a separate PR? Migrating the ESLint config to the new format is sometimes quite involved.

---

`eslint-config-standard` hasn't received updates in 2 years and doesn't support ESLint v9. It seems like maintenance has [stalled out](https://github.com/standard/standard/issues/1948) and some work is happening on the [`neostandard` fork](https://github.com/neostandard/neostandard).

We'll either have to switch to the fork or migrate to something new. I mentioned elsewhere on this PR already, but I like ESLint's recommended config over standard.

I would still suggest updating this separately. My goal with this PR was to get ESLint running correctly so the repository remains linted. If you know what configuration you want to use for ESLint v9, I may be able to help with that update too.

---

yes, this is something I'd like to discuss as well. I'm happy to use Neostandard, though I'm also open to using tools like Biome, which include both linting and formatting.